content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+
+    -   To publicly represent the committee as the direct link to
+        external groups;
+
+    -   To represent the undergraduate committee at meetings of the WiCS
+        Committee;
+
+    -   To represent the undergraduate committee at meetings of any
+        other external organization; to which the commiittee is
+        accountable, when appropriate, such as the Mathematics Society;
+
+    -   To secure funding for the committee's endeavours in
+        collaboration with the Finance Director.
+
+3.  The duties of the Secretary are:
+
+    -   To take minutes at all meetings;
+
+    -   To call meetings;
+
+    -   To represent the undergraduate committee at meetings of the WiCS
+        Committee;
+
+    -   To build consensus among members in the decision-making process;
+
+    -   To ensure all members are comleting assigned resposibilities;
+
+    -   To maintain the non-financial records of the committee.
+
+4.  The duties of the Finance Director are:
+
+    -   To process financial paperwork;
+
+    -   To maintain accurate budgets and financial records;
+
+    -   To secure funding for the endeavours of the committee, in
+        collaboration with the Chair;
+
+    -   To represent the organization in a financial capacity, in all
+        relevant meetings.
+
+5.  The duties of the Office Manager are:
+
+    -   To maintain the Office Policy;
+
+    -   To train and manage office staff;
+
+    -   To ensure the office is a clean, orderly, and respectful space;
+
+    -   To create and manage the schedule of office hours;
+
+    -   To ensure the office is equipped with adequate supplies.
+
+6.  The duties of the System Administrator are:
+
+    -   To chair the Systems Committee ("wics-sys");
+
+    -   To maintain the committee's systems, including office terminals
+        and servers;
+
+    -   To maintain the official website, wics.uwaterloo.ca;
+
+    -   To be the authority on matters regarding usage of systems.
+
+7.  The duties of the Big CSters Director are:
+
+    -   To maintain the Big CSters community through useful and
+        inclusive programming, such as workshops and discussion circles;
+
+    -   To run the Big CSters mentoring program, holding an absolute
+        minimum of two events per term;
+
+    -   To administer the CSters mailing list.
+
+8.  The duties of the Diretor of Publicity are:
+
+    -   To publicize events and other endeavours of the committee
+        through physical and electronic means;
+
+    -   To design all media circulated by the committee including:
+        posters, social media profiles and the official website.
+
+# Procedures #
+
+Any changes to procedures must be proposed with at least one week's
+notice, and ratified at an official meeting of the undergraduate
+committee.
+
+## Policy ##
+
+1.  Code of Conduct\
+    The Code of Conduct outlines expected behaviour for all members of
+    the community at any event affiliated with the undergraduate
```

Comment:
CoC doesn't just apply to events; it applies to any committee space and any business pertaining to the committee. Per scope:

We expect all community participants (participants, organizers, sponsors, and other guests) to abide by this Code of Conduct in all community venues—online and in-person—as well as in all one-on-one communications pertaining to community business.

so this means office, any communications (written/electronic) including mailing lists/IRC, at meetings, talking about committee business one-on-one, at events, etc.